main.tf

```diff
@@ -130,3 +130,25 @@ resource "aws_cloudwatch_metric_alarm" "database_memory_free" {
   ok_actions                = ["${var.ok_actions}"]
   insufficient_data_actions = ["${var.insufficient_data_actions}"]
 }
+
+resource "aws_cloudwatch_metric_alarm" "database_cpu_credits" {
+  // This results in 1 if instance_type starts with "db.t", 0 otherwise.
+  count               = "${replace(replace(var.instance_type, "/^db\\.[^t].*/", "0"), "/^db\\.t.*$/", "1")}"
```

Comment:
I have yet to use this interpolation method, but have you tried what's described here? https://www.hashicorp.com/blog/terraform-0-8/#conditional

```hcl
variable "env" { default = "development" }

resource "aws_instance" "foo" {
  count = "${var.env == "production" ? 1 : 0}"
}
```

Perhaps we can use `instance_type` in the conditional to go down one branch vs. the other with `contains()` or `split()`? Mostly just trying to make it easier to digest for someone who tries to read this a few weeks or month down the road.

I'm also cool with making this module Terraform 0.8+ only.